/**
 * Copyright Soramitsu Co., Ltd. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */

syntax = "proto3";
package iroha.network.proto;

import "block.proto";

message BlockRequest {
  uint64 height = 1;
}

service Loader {
  rpc retrieveBlocks (BlockRequest) returns (stream iroha.protocol.Block);
  rpc retrieveBlock (BlockRequest) returns (iroha.protocol.Block);
}
